gini/gini-vision-lib-android
# Feature multi page jpeg exif and compression (after #207)

JPEG compression and EXIF handling was not done for all multi-page cases.

* Camera Screen
  * Taken pictures are compressed and their EXIF is updated before proceeding
  * If multi-page is enabled then imported JPEGs are compressed and their EXIF is updated before proceeding
* Review Screen (used only if multi-page is not enabled)
  * Only imported JPEG is compressed and its EXIF updated
* "Open with" JPEGs
  * Only multi-page import JPEGs are compressed and their EXIF updated

### How to test
Run one of the example apps and check the EXIF of the JPEGs saved in the example app's external storage.
